botstrap.py

```diff
@@ -122,10 +182,29 @@ def create_webhook(name: str, channel_id_: int, client: DiscordClient) -> str:
 
         config_str += f"roles_{role_name}={role_id}\n"
 
-    all_channels, all_categories = get_all_channels_and_categories(guild_id=GUILD_ID, client=discord_client)
+    all_channels, all_categories = discord_client.get_all_channels_and_categories()
 
     config_str += "\n#Channels\n"
 
+    rules_channel_id = all_channels[RULES_CHANNEL_NAME]
+    announcements_channel_id = all_channels[ANNOUNCEMENTS_CHANNEL_NAME]
+
+    discord_client.upgrade_server_to_community_if_necessary(rules_channel_id, announcements_channel_id)
+
+    create_help_channel = True
+
+    python_help_channel_id = all_channels[PYTHON_HELP_CHANNEL_NAME]
```

Comment:
If the server contains a forum channel it won't be in the template, so you can't assume this channel exists. The way it was before was fine IMO.

Reply:
Euh, idk what happened here tbh.

Oui, i need to bring it back.

Reply:
dace921097e56dc39d6643f384d1ca08befc9560